benchmark/cluster/README.md

```diff
@@ -0,0 +1,54 @@
+# Cluster Training Benchmark
+
+## Setup
+
+- Platform
+  - Kubernetes: v1.6.2
+  - Linux Kernel: v3.10.0
+
+- Resource
+  - CPU: 10 Cores per Pod
+  - Memory: 5GB per Pod
+
+- Docker Image
+
+  We use different base Docker Image to run the benchmark on Kubernetes:
+  - PaddlePaddle v2: paddlepaddle/paddle:latest
+  - PaddlePaddle Fluid: paddlepaddle/paddle:latest
+  - TensorFlow: tensorflow/tensorflow:latest
+
+- Model
+  A digits recognize model and MNIST dataset is used in this benchmark.
```

Comment:
I think this model is too small. Maybe [vgg-16](https://github.com/PaddlePaddle/Paddle/blob/develop/python/paddle/v2/fluid/tests/book/test_image_classification_train.py#L90) (probably around 500MB) is a better choice.

Reply:
Done.